README.md

```diff
@@ -144,7 +211,13 @@ PYENV 3.10 venv1
 
 ### `@task.venv` decorator
 
-TODO! Write the decorator, then fill out docs!
+The `@task.venv` decorator wraps the [ExternalPythonOperator](https://airflow.apache.org/docs/apache-airflow/stable/howto/operator/python.html#externalpythonoperator). The decorator does a few things:
+1. It assigns the decorated function as the `ExternalPythonOperator`'s callable
+2. It assigns the string passed to the decorator to the `ExternalPythonOperator`'s `python` parameter.
```

Comment:
That's not quite what it does -- it actually looks up full path form the environment variables that the buildkit puts in to the image.

Maybe we just say something to the tune of "it looks up the absolute path of the virtual env ..."